.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules/
 # Build directory
 **/dist
 **/public
+.DS_Store
```

Comment:
for some reason. it's still here... =)

Reply:
You mean .DS_Store file somewhere in the repo?

Reply:
<img width="640" alt="image" src="https://github.com/fluencelabs/js-client/assets/59872966/a32afb4a-86c7-418d-b7e4-223323de09e7">
Cant find it

Reply:
no, sorry I mean fucking new line missing =)

Reply:
k, fixed